cmd/ocm/create/cluster/cmd.go

```diff
@@ -377,6 +394,68 @@ func getVersionOptionsWithDefault(connection *sdk.Connection, channelGroup strin
 	return
 }
 
+func getSubscriptionTypeOptions(connection *sdk.Connection) ([]arguments.Option, error) {
+	options := []arguments.Option{}
+	billingModels, err := getBillingModels(connection)
+	if err != nil {
+		return options, err
+	}
+	for _, billingModel := range billingModels {
+		//Standard billing model should always be the first option
+		if billingModel.Id() == StandardSubscriptionType {
+			standardOption := arguments.Option{
+				Value: fmt.Sprintf("%s (%s)", billingModel.Id(), billingModel.Description()),
+			}
+			if len(options) == 0 { // nil or empty slice or after last element
+				options = append(options, standardOption)
+			} else {
+				options = append(options[:1], options[0:]...)
+				options[0] = standardOption
+			}
+		} else {
+			options = append(options, arguments.Option{
+				Value: fmt.Sprintf("%s (%s)", billingModel.Id(), billingModel.Description()),
+			})
+		}
+	}
+	return options, nil
+}
+
+func getSubscriptionTypeIdFromDescription(connection *sdk.Connection, value string) string {
+	billingModels, _ := getBillingModels(connection)
+	for _, billingModel := range billingModels {
+		if value == fmt.Sprintf("%s (%s)", billingModel.Id(), billingModel.Description()) {
+			return billingModel.Id()
+		}
+	}
+	return ""
+}
+
+func getBillingModel(connection *sdk.Connection, billingModelID string) (*amv1.BillingModelItem, error) {
```

Comment:
We can make this function reusable (as well as getBillingModels) by moving them to its own package under pkg/billing, for example.

This logic doesn't really belong to cmd/ocm/create/cluster/cmd.go

Reply:
Moved to the billing package